src/server/pegasus_server_impl.cpp

```diff
@@ -1434,6 +1446,14 @@ ::dsn::error_code pegasus_server_impl::start(int argc, char **argv)
             nullptr, // TODO: the tracker is nullptr, we will fix it later
             [this]() { update_server_rocksdb_statistics(); },
             _update_rdb_stat_interval);
+
+        _update_server_rdb_limiter_stat = ::dsn::tasking::enqueue_timer(
+            LPC_REPLICATION_LONG_COMMON,
+            nullptr,
+            [this]() {
+                update_server_rocksdb_rate_limiter_statistics(_update_rdb_stat_short_interval);
```

Comment:
`_update_rdb_stat_short_interval ` is a member variable, not need to pass it as a parameter.

Reply:
`update_server_rocksdb_rate_limiter_statistics` is a static method which can‘t use non-static member `_update_rdb_stat_short_interval`

Reply:
You can use `FLAGS_update_rdb_stat_short_interval ` directly, then you can gain benifit of a future feature: dynamic modify configs.